.github/workflows/test.yml

```diff
@@ -14,8 +14,6 @@ jobs:
           registry-url: "https://registry.npmjs.org"
       - name: Install deps
         run: yarn
-      - name: Check types
-        run: yarn check-types
```

Comment:
that is checking types in tests as well

Reply:
The next step `yarn build` does the type checking